### PR TITLE
Update readme file

### PR DESCRIPTION
**Pull request by kirch1**
I edited the readme file

### CODE REVIEW

**Comment by davejm8** [comment]
This code is terrible

**Comment by kirch1** [comment]
> 

Reporting you to HR

File: README.md [hunk 1]
```diff
@@ -1 +1,5 @@
-This is really fun!
\ No newline at end of file
+# This is really fun!
```
Comment (kirch1) [inline README.md:1]: hi

Comment (radsy207) [inline README.md:1]: asfgsdfhs

Comment (kirch1) [reply README.md:1]: @radsy207

Comment (davejm8) [reply README.md:1]: asfhsdfhjdfgj

**Review by davejm8 — APPROVED** [review]
It was super good

**Pull request merged**